relay/README.md

```diff
@@ -84,61 +65,79 @@ Scene 2:
 - QmOne tries to connect via relaying, because it shares this transport with QmTwo.
 - A lively and prolonged dialogue ensues.
 
-
 ## Addressing
 
-`/p2p-circuit` multiaddrs don't carry any meaning of their own.
-They need to encapsulate a `/p2p` address, or
-be encapsulated in a `/p2p` address, or both.
+`/p2p-circuit` multiaddrs don't carry any meaning of their own. They need to encapsulate a `/p2p` address, or be encapsulated in a `/p2p` address, or both.
+
+As with all other multiaddrs, encapsulation of different protocols determines which metaphorical tubes to connect to each other.
 
-As with all other multiaddrs, encapsulation of different protocols
-determines which metaphorical tubes to connect to each other.
+A `/p2p-circuit` circuit address, is formated following:
+
+`/p2p-circuit[<relay peer multiaddr>]<destination peer multiaddr>`
+
+This opens the room for multiple hop relay, where the first relay is encapsulated in the seconf relay multiaddr, such as:
+
+`/p2p-circuit/p2p-circuit/<first relay>/<first hop multiaddr>/<destination peer multiaddr>`
```

Comment:
To me it makes a lot more sense to do something like - `/p2p-circuit<first relay>/<first hop multiaddr>/p2p-circuit<second relay>/<second hop multiaddr>/<destination peer multiaddr>`